custom_components/adaptive_lighting/manifest.json

```diff
@@ -7,6 +7,6 @@
   "documentation": "https://github.com/basnijholt/adaptive-lighting#readme",
   "iot_class": "calculated",
   "issue_tracker": "https://github.com/basnijholt/adaptive-lighting/issues",
-  "requirements": ["ulid-transform"],
+  "requirements": ["ulid-transform","aiohue-BenoitAnastay"],
```

Comment:
Why can't `aiohue` be used?

Reply:
Scene update wasn't implemented so I made my own and my PR was discarded because of depreciation of V1 API

I'll check if it was added to the main projet and update this PR.
I also need to fix the fact that the scenes arent updated when all lights are off